alea/template_source.py

```diff
@@ -2,62 +2,147 @@
 import warnings
 import logging
 
-import blueice
 import numpy as np
 from scipy.interpolate import interp1d
+from blueice import HistogramPdfSource
 from inference_interface import template_to_multihist
 
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(level=logging.debug)
```

Comment:
Ok for testing but we should revert this before merging

Reply:
Reverted.